### PR TITLE
Modified likes functionality 

### PR DESCRIPTION
**Pull request by patnunes**
Modified user_has_liked_tweet() and like_a_tweet()
Added is_liked() to views.py to return info to FE.

### CODE REVIEW

**Comment by Nwitz** [comment]
This looks good to me and has worked, @rickyelopez , please confirm

File: tweeter/main_app/tweets.py [hunk 1]
```diff
@@ -2,10 +2,11 @@
 from datetime import datetime
 from django.core.serializers.json import DjangoJSONEncoder
 import json
+from django.db import connection
 
 
 def save_user_tweets(tweetData):
-    # TODO: setup the likes counter once the likes feature has been implemented
+    #TODO: setup the likes counter once the likes feature has been implemented
```
Comment (Nwitz) [inline tweeter/main_app/tweets.py]: Has this been completed? do we still need the TODO?

Comment (patnunes) [reply tweeter/main_app/tweets.py]: Yes, the likes counter is being incremented

**Comment by rickyelopez** [comment]
resolves #125

**Pull request merged**